daemon/cmd/daemon_main.go

```diff
@@ -1370,6 +1373,24 @@ func initEnv(cmd *cobra.Command) {
 			option.EgressMultiHomeIPRuleCompat,
 		)
 	}
+
+	if option.Config.InstallNoConntrackIptRules {
+		// InstallNoConntrackIptRules can only be enabled in direct
+		// routing mode as in tunneling mode the encapsulated traffic is
+		// already skipping netfilter conntrack.
+		if option.Config.Tunnel != option.TunnelDisabled {
+			option.Config.InstallNoConntrackIptRules = false
+			log.Infof("%s requires the agent to run in direct routing mode. Disabling it.", option.InstallNoConntrackIptRules)
```

Comment:
This should be a warning. An option being explicitly ignored should be a notable event.